pysat/instruments/pysat_testmodel.py

```diff
@@ -101,14 +94,16 @@ def load(fnames, tag=None, inst_id=None, num_samples=None):
 
     # Fake 3D data consisting of values between 0 and 21 everywhere
     dummy1 = np.mod(data['uts'] * data['latitude'] * data['longitude'], 21.0)
-    data['dummy1'] = (('time', 'latitude', 'longitude'), dummy1)
+    data['dummy1'] = (('time', 'latitude', 'longitude'), dummy1.data)
```

Comment:
The multidimensional data sets could use a description of what they are trying to simulate.